cmd/bnscli/clitests/sign_send_tokens.test

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+if [ -z ${TM_ADDR} ]
+then
+	# TM_ADDR must be either set by the host or it will be configured by
+	# the test runner to use tendermint mock server.
+	>&2 echo "TM_ADDR must be set in order for this test to run."
```

Comment:
Note that when I run it, I get this line,
And also the `sign_send_tokens.test.gold` is empty (as it expects this failure).

If you want to include any `send` tests (I like it), please start up a local node for the tests to run against.
There are example scripts in iov-core and the wallet-demo and other repos.

Reply:
The `cli_main.go` is running a fake server. I think this is enough for a fast test.

I will think about providing a real server. I know this is way more work with all the configuration, recent `bnsd` binary is required and the process cannot be controlled (sleep here and there to ensure the server is ready :see_no_evil:). 

I think that for now, using a fake tendermint server to have an instant test feedback and an easy way to write those tests is worth the tradeof. `bnscli` is meant to be used and we can always run a command manually to check it. This is a very simple thing to do, while to automate it takes a lot of effort.